app/models/order.rb

```diff
@@ -13,22 +13,23 @@ class Order < ActiveRecord::Base
 
   delegate :name, to: :order_status, prefix: true
 
-  scope :pending, -> { where(order_status_id: 1) }
+  scope :pending, -> { where(order_status_id: OrderStatus.find_by(name: "Pending")) }
```

Comment:
I'm going to do something along those lines. I have bigger priorities on this project at the moment, but I plan to rewrite the way OrderStatuses are handled throughout the app. Annie and I discussed a better solution yesterday.